src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int figureCount = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        double maxSide = 10;
+        int figureNumber = random.nextInt(figureCount);
+        double firstSide = random.nextDouble() * maxSide;
+        double secondSide = random.nextDouble() * maxSide;
+        double thirdSide = random.nextDouble() * maxSide;
+        switch (figureNumber) {
+            case 0 :
+                return new Square(firstSide);
```

Comment:
create a private method for prepare Figures
```suggestion
                return prepareSquare();
```

Reply:
not fixed